jscomp/others/belt_MutableSetInt.mli

```diff
@@ -30,95 +30,605 @@
   and identity is not needed(using the built-in one)
 
   **See** [`Belt.MutableSet`]()
+
+  This module is [Belt.MutableSet](mutable-set) specialized with key type to be a `int` type.
+  It is more efficient in general, the API is the same with [Belt.MutableSet](mutable-set) except its key type is fixed, and identity is not needed (using the built-in one).
 *)
 
 
 # 38 "others/setm.cppo.mli"
 type value = int
-  
+(**
+  ```res prelude
```

Comment:
I am not entirely sure what to do with our `prelude` metas... probably better to not use a `(**` and make it a "normal comment" with a `(*` instead (just to keep the prelude around without it showing in the output)